exercises/poker/src/example/java/Poker.java

```diff
@@ -0,0 +1,142 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Poker {
```

Comment:
This class is looking a lot better now! :) But it is quite long with the two nested classes, Card and Hand, in it. I would suggest extracting these into their own files. That would make each file more readable :)